### PR TITLE
Mainnet node release and cargo 19 Jan

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about 5.1.3 node release on mainnet and cargo release on mainnet and testnet.

## Changes

Both release notes, both downloads, variables

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

File: source/mainnet/net/resources/release-notes.rst [hunk 1]
```diff
@@ -11,6 +11,12 @@ Release Notes - Testnet
 
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+January 19, 2023
+
+Cargo concordium 2.6.0
+---------------------------
+
+
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:19]: This should be removed for now, right?

Comment (abizjak) [reply source/mainnet/net/resources/release-notes.rst:19]: I have not checked it out locally. I just see it in the diff here (on github) that's why I commented.

But if it is gone then all is good. :)

Comment (dg-concordium) [reply source/mainnet/net/resources/release-notes.rst]: I forgot it was in the testnet release notes too. Now it is gone.

**Review by mh-concordium — APPROVED** [review]
Looks good.

**Review by mh-concordium — APPROVED** [review]

**Pull request merged**